forge-gui-mobile/src/forge/adventure/scene/QuestLogScene.java

```diff
@@ -31,9 +31,9 @@ private QuestLogScene() {
         scrollWindow = ui.findActor("scrollWindow");
         root = ui.findActor("questList");
         detailRoot = ui.findActor("questDetails");
-        abandonQuestButton = ui.findActor("abandonQuest");
-        trackButton = ui.findActor("trackQuest");
-        backToListButton = ui.findActor("backToList");//Controls.newTextButton("Quest List");
+        abandonQuestButton = Controls.newTextButton("Abandon Quest");
```

Comment:
Is Adventure mode not hooked up with Internationalization at all?

Reply:
It is, and that's a step I need to return to. I'll get that wired up for these changes.